backend/README.md

```diff
@@ -0,0 +1,83 @@
+# IdeaDog Server
+[![Build status](https://ci.appveyor.com/api/projects/status/jramqn9wwm2qk13j/branch/master?svg=true)](https://ci.appveyor.com/project/Ostoyae/ideadog-server/branch/master)
+
+## Description :bulb:
+
+*Currently the back-end and [front-end](https://github.com/bdbaraban/ideadog) of IdeaDog live in seperate repos, to reduce project clutter.*
+
+IdeaDog is a social web application for sharing ideas. We've all had those sudden bursts of inspiration where we go - "wow, that is simply a _great_ idea" - but know we will never follow up. Now, instead of losing those ideas, share them with the world!
```

Comment:
Let's not repeat ourselves in the README - remove duplicate content and keep the `backend` README exclusively dedicated to describing the backend.

Remove:
```
IdeaDog is a social web application for sharing ideas. We've all had those sudden bursts of inspiration where we go - "wow, that is simply a _great_ idea" - but know we will never follow up. Now, instead of losing those ideas, share them with the world!

Attach files by dragging & dropping, selecting or pasting them.


Why statically-typed, you ask? Well, the front-end of the application is built in React, with TypeScript, while the back-end runs on Rust. TypeScript + Rust = The Ultimate Statically-Typed Web App!

Development of IdeaDog has only just begun. Stay tuned, there is much, much, much more to come!
```

```

**Front-End**:
[Source](https://github.com/bdbaraban/ideadog)

| Tool/Library             | Version |
| ------------------------ | ------- |
| [TypeScript](https://www.typescriptlang.org/) | ^3.5.1  |
| [React](https://reactjs.org/) | ^16.8.6 |
| [Material UI](https://material-ui.com/) | ^4.0.2  |
| [Navi](https://frontarm.com/navi/en/) | ^0.12.7 |
| [@dwqs/react-virtual-list](https://www.npmjs.com/package/@dwqs/react-virtual-list) | ^1.0.0  |
| [clipboard-copy](https://www.npmjs.com/package/clipboard-copy) | ^3.0.0  |

View the complete list of front-end dependencies in the corresponding [package.json](./frontend/package.json).
```